accounts/forms.py

```diff
@@ -9,7 +9,7 @@
 class CustomUserCreatationForm(UserCreationForm):
     class Meta:
         model = get_user_model()
-        fields = ('email', 'password1', 'password2')
+        fields = ('full_name','email','website', 'password1', 'password2')
```

Comment:
We do not need `full_name` , and `website` fields in User Creation Form, please remove it from here.